src/tools/dev/masonry/bootstrap_visit.py

```diff
@@ -258,176 +254,21 @@ def steps_package(opts,build_type,ctx):
-    pl = plistlib.readPlistFromString(res)
-    uuid = pl["notarization-upload"]["RequestUUID"]
-
-    print "[uuid: %s]" % uuid
-
-    # Check status of notarization request
-    cmd = "xcrun altool --notarization-info %s" % uuid
-    cmd += " --username %s" % opts["notarize"]["username"]
-    cmd += " --password %s" % opts["notarize"]["password"]
-    cmd += " --output-format xml"
-    print "[exe: %s]" % cmd
-
-    status = "in progress"
-    while status == "in progress":
-        time.sleep(30)
-        p = subprocess.Popen(cmd, **kwargs)
-        res = p.communicate()[0]
-        pl = plistlib.readPlistFromString(res)
-        status = pl["notarization-info"]["Status"]
-        status = status.strip()
-        print "[status: %s]" % status
-
-    ###################################
-    # Staple notarization ticket to app bundle
-    ###################################
-
-    if status == "success":
-        cmd = "xcrun stapler staple %s" % visit_app
-        print "[exe: %s]" % cmd
-        p = subprocess.Popen(cmd, **kwargs)
-        res = p.communicate()[0]
-        print "[stapler: %s]" % res
-
-        # Create new DMG with stapled containing notarized app
-        dmg_stapled = pjoin(notarize_dir, "VisIt.stpl.dmg")
-        cmd = "hdiutil create -srcFolder %s -o %s" % (src_folder, dmg_stapled)
-        print "[exe: %s]" % cmd
-        p = subprocess.Popen(cmd, **kwargs)
-        res = p.communicate()[0]
-        print "[hdiutil: %s]" % res
-
-        dmg_release = pjoin(notarize_dir, "VisIt-%s.dmg" % opts["version"])
-        cmd = "hdiutil convert %s -format UDZO -o %s" % (dmg_stapled, dmg_release)
-        print "[exe: %s]" % cmd
-        p = subprocess.Popen(cmd, **kwargs)
-        res = p.communicate()[0]
-        print "[hdiutil:convert: %s]" % res
-    else:
-        raise RuntimeError("Notarization Failed: %s" % res)
+    if opts["platform"] == "osx":
+        a_notarize_visit  = "notarize_visit." + build_type
+        ctx.actions[a_notarize_visit] = notarize(build_dir=opts["build_dir"],
+                                                 build_type=build_type.lower(),
+                                                 build_version=opts["version"],
+                                                 build_arch=opts["arch"],
+                                                 entitlements=opts["entitlements"],
+                                                 cert=opts["cert"],
+                                                 bundle_id=opts["notarize"]["bundle_id"],
+                                                 username=opts["notarize"]["username"],
+                                                 password=opts["notarize"]["password"],
+                                                 asc_provider=opts["notarize"]["asc_provider"],
+                                                 description="notarizing visit")
+        ctx.triggers["build"].append(a_notarize_visit)
 
 def steps_sanity_checks(opts,build_type,ctx):
```

Comment:
Happened to see the name of this function and it occurs to me to mention that in my [*inclusive language*](https://inclusivenaming.org) work, I've learned that the phrase *sanity check* is one of [those terms](https://developers.google.com/style/inclusive-documentation) that we use routinely that has negative connations for neuro-diverse populations. Not suggesting we need to change this but simply using it as an *inclusive moment*.